### ISSUE TITLE
Lightspeed plugin: Enable accessibility checks and fix remaining a11y issues

### ISSUE THREAD

**Issue by christoph-jerolimov** [redhat-developer/rhdh-plugins#3588]
The lightspeed plugin has playwright tests that include a bunch of accessibility assertions.

The tests runs `runAccessibilityTests` from `@workspaces/lightspeed/e2e-tests/utils/accessibility.ts`, but failures are ignored at the moment.

1. Remove the `options` parameter with the `skipFailures` flag and run the expect statement always.
2. Run the playwright tests via `CI=1 yarn test:e2e` and list the left over issues.
3. Try to resolve these issues in the code base. When unsure, or not all issues can be removed, create a PR anyway.
